lib/rubocop/cop/style/send.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for the use of the send method.
+      class Send < Cop
+        MSG = 'Prefer `BasicObject#__send__` or `Object#public_send` to `send`,
+               as `send` may overlap with existing methods.'
```

Comment:
As it stands the message is confusing. I'd suggest dropping the second line completely, as otherwise you'd have to explain the value of `public_send` and this would make the message way too long.